synfig-studio/src/gui/workarea.cpp

```diff
@@ -2185,15 +2194,22 @@ studio::WorkArea::set_cursor(const Glib::RefPtr<Gdk::Cursor> &x)
 	//!Check if the window we want draw is ready
 	Glib::RefPtr<Gdk::Window> draw_area_window = drawing_area->get_window();
 	if (!draw_area_window) return;
-	draw_area_window->set_cursor(x);
+
+	// do not change cursor in drag window mode
```

Comment:
Please change to
`	// do not change cursor while in drag window mode; postpone this change until after stopping window dragging`

Or something like that :)